src/Raven.Studio/typescript/viewmodels/database/settings/databaseSettings.ts

```diff
@@ -0,0 +1,526 @@
+                    this.editModeHasBeenEntered = true;
+                    this.viewMode("editMode");
+                    this.initCategoryGrid();
+                    this.setupEditMode(false);
+                });
+        }
+    }
+    
+    private setupEditMode(reset: boolean = true) {
+        this.dirtyFlag().reset();
+        
+        if (reset) {
+            this.categoriesGridController().reset(false);
+        }
+        
+        this.setCategory(this.selectedCategory() || this.allCategoryNames()[0]);
+        this.computeEntriesToShow();
+       
+        this.allEntries().forEach(entry => {
+            entry.showEntry(this.shouldShowEntry(entry));
+
+            if (entry instanceof models.databaseEntry) {
+                entry.override.subscribe((override) => {
+                    
+                    const category = this.categoriesInfo().find(x => x.name() === entry.data.Metadata.Category);
+                    const currentCustomizedCount = category.customizedEntriesCount();
+                    if (override) {
+                        category.customizedEntriesCount(currentCustomizedCount+1)
+                    } else {
+                        category.customizedEntriesCount(currentCustomizedCount-1)
+                    }
+                    
+                    this.categoriesGridController().reset(false);
+                    this.setCategory(entry.data.Metadata.Category);
+                });
+                
+                entry.customizedDatabaseValue.throttle(300).subscribe(() => {
+                    this.categoriesGridController().reset(false);
+                });
+            }
+        });
+    }
+
+    private initializeDirtyFlag() {
+        
+        const hasAnyDirtyField = ko.pureComputed(() => {
+            let anyDirty = false;
+            
+            this.allEntries().forEach(entry => {
+                if (entry instanceof models.databaseEntry && entry.entryDirtyFlag().isDirty()) {
+                    anyDirty = true;
+                }
+            });
+            
+            return anyDirty;
+        });
+        
+        this.dirtyFlag = new ko.DirtyFlag([
+            hasAnyDirtyField
+        ], false, jsonUtil.newLineNormalizingHashFunction);
+    }
+
+    readonly howToReloadDatabaseHtml = `<h4 class="">There are two ways to reload the database:</h4>
+                                        <ul>
+                                            <li>
+                                                <small>
+                                                    Disable and then enable the database from the databases-list-view in the Studio.<br>
+                                                    This will reload the database on all the cluster nodes immediately.
+                                                </small>
+                                            </li>
+                                            <li class=\"margin-top margin-top-sm\">
```

Comment:
do we need to escape " ? We are  in ` context

Reply:
removed